{
    "error": "HTTPSConnectionPool(host='bugzilla.andrew.cmu.edu', port=443): Max retries exceeded with url: /cgi-bin/cvsweb.cgi/src/sasl/plugins/digestmd5.c?rev=1.171&amp;amp;content-type=text/x-cvsweb-markup (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f01d40a3f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
}